code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1877,4 +1877,30 @@ Departamental Swimsuits, for general use
 	rolled_down = 0
 	rolled_sleeves = 0
 	starting_accessories = null
-	item_icons = list()
\ No newline at end of file
+	item_icons = list()
+
+//Vorrarkul: Melanie Farmer
+/obj/item/clothing/under/fluff/slime_skeleton
```

Comment:
Should probably make it indigestible as requested in the app. Making digest_act return null should do it.

Reply:
Oh god damn it, I wanted to do that but when the species check didn't work as expected, I got distracted and forgot about it... Apologies, gonna fix in a few.